dg/media/coco/app/scripts/auth_offline_backend.js

```diff
@@ -26,13 +26,13 @@ define([
   }
   
   // if u, p matches that in user table, sets login state = true 
-  var login = function(username, password, language, partner_name){
+  var login = function(username, password, language, partner_name, partner_id, user_id){
```

Comment:
cant we have any one? both partner id and name need not be passed every time. Maybe we can get name from id or vice-versa.